.bento/archive.json

```diff
@@ -1,51 +0,0 @@
-{
-  "bandit": {
-    "violations": {
-      "4e0db4785748aff9799910743970a07b": {
-        "tool_id": "bandit",
-        "check_id": "start-process-with-a-shell",
-        "path": "semgrep/setup.py",
-        "message": "Starting a process with a shell, possible injection detected, security issue.",
-        "severity": 2,
-        "syntactic_context": "os.system(  # nosem: contrib.dlint.dlint-equivalent.insecure-os-exec-use\n    os.path.join(repo_root, \"release-scripts/ubuntu-release.sh\")"
```

Comment:
Do we still need the `nosem` in the source files, if we're now excluding `setup.py`?

Reply:
nm, I read to the end